docs/_includes/api/create_database.html

```diff
@@ -33,23 +33,14 @@
 
 * `size`: Amount in MB to request for storage, which you will need if you are storing >5MB in order to [avoid storage limit errors on iOS/Safari](/errors.html#not_enough_space). Persistent views use a seperate database per view which will also request storage space so keep in mind the upper storage limits when using persistent views on iOS.
 
-**SQLite Plugin-only options:**
-
-Any options given to the PouchDB constructor will be passed verbatim to the SQLite Plugin.  Older versions of PouchDB supported only the following, which may be of use:
-
-* `location`: Where to store data on iOS, which may affect iTunes/iCloud backup, and thus whether or not your app gets rejected by Apple. See [SQLite Plugin documentation](https://github.com/brodysoft/Cordova-SQLitePlugin/#opening-a-database) and [iOS data storage guidelines](https://developer.apple.com/icloud/documentation/data-storage/index.html).
-* `createFromLocation`: Use a pre-populated database, so you can package it with your app and users don't have to wait for it to load. See the [SQLite Plugin documentation](https://github.com/brodysoft/Cordova-SQLitePlugin/#pre-populated-database).
-* `androidDatabaseImplementation`: `1` to use the new `sqlite4java` implementation, `2` to use the older version based on the native Android APIs. Defaults to `1`. See the [SQLite Plugin documentation](https://github.com/litehelpers/Cordova-sqlite-storage#android-sqlite-implementation).
-
 **Notes:**
 
 1. In IndexedDB and WebSQL, PouchDB will use `_pouch_` to prefix the internal database names. Do not manually create databases with the same prefix.
 2. When acting as a client on Node, any other options given will be passed to [request][].
-3. When using the `'leveldb'` adapter (the default on Node), any other options given will be passed to [levelup][]. The storage layer of leveldb can be replaced by passing a level backend factory (such as [MemDOWN][]) as `options.db`. The rest of the [supported levelup options are documented here][levelup_options].
+3. When using the `'leveldb'` adapter (the default on Node), any other options given will be passed to [levelup][].
```

Comment:
I would like to deprecate `db` as well. It's an internal detail of `pouchdb-adpater-leveldb-core`, and furthermore there are no LevelDOWN adapters I'm aware of that do anything interesting with PouchDB other than the ones we ourselves maintain (such as LocalStorageDOWN, MemDOWN, and FruitDOWN). The e.g. RedisDOWN/RiakDOWN/etc. adapters were interesting experiments but ultimately pointless because perf sucks and they didn't solve the problem people hoped they would solve (PouchDB replication without CouchDB).